spec/acceptance/snmp_spec.rb

```diff
@@ -5,14 +5,17 @@
   client_package = 'net-snmp-utils'
   service_config = '/etc/snmp/snmpd.conf'
   service_varconfig = '/var/lib/net-snmp/snmpd.conf'
+  modified_snmpd_options = '-LS0-4d'
 when 'Debian'
   client_package = 'snmp'
   service_config = '/etc/snmp/snmpd.conf'
   service_varconfig = '/var/lib/snmp/snmpd.conf'
+  modified_snmpd_options = '-Ls4d -Lf /dev/null -u Debian-snmp -g Debian-snmp -I -smux,mteTrigger,mteTriggerConf -f'
```

Comment:
This looks the default value for snmpd options. Am i wrong ?

The goal of the this acceptance test if to run the the snmpd process with defferent options and check that modified options are used.

Reply:
default is ```-Lsd -Lf /dev/null -u Debian-snmp -g Debian-snmp -I -smux,mteTrigger,mteTriggerConf -p /run/snmpd.pid'```

Reply:
OK